tests/pr_test.go

```diff
@@ -14,9 +14,14 @@ const defaultExampleTerraformDir = "examples/default"
 
 func setupOptions(t *testing.T, prefix string, dir string) *testhelper.TestOptions {
 	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
-		Testing:       t,
-		TerraformDir:  dir,
-		Prefix:        prefix,
+		Testing:      t,
+		TerraformDir: dir,
+		Prefix:       prefix,
+		IgnoreUpdates: testhelper.Exemptions{ // Ignore for consistency check
```

Comment:
why do we need to ignore update on the sleep_time?

Reply:
the upgrade test was catching the change in name of the sleep_time block.
```Resource(s) identified to be updated Name: sleep_time Address: time_sleep.sleep_time Actions: [update]```